frontend/packages/ux-editor/src/hooks/mutations/useAddItemToLayoutMutation.ts

```diff
@@ -18,6 +19,7 @@ export const useAddItemToLayoutMutation = (org: string, app: string, layoutSetNa
   const { layout, layoutName } = useFormLayoutsSelector(selectedLayoutWithNameSelector);
   const formLayoutsMutation = useFormLayoutMutation(org, app, layoutName, layoutSetName);
   const appAttachmentMetadataMutation = useAddAppAttachmentMetadataMutation(org, app);
+  const getLayoutSets = useLayoutSetsQuery(org, app);
```

Comment:
Du kan bruke destrukturering her til å få ut `layoutSets` direkte:
```suggestion
  const { data: layoutSets } = useLayoutSetsQuery(org, app);
```